src/test/kotlin/org/reactome/cli/ReactomeCliTest.kt

```diff
@@ -0,0 +1,17 @@
+package org.reactome.cli
+
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.cio.CIO
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class ReactomeCliTest {
+
+    @Test
+    fun `Should return response for one protein`() {
+        val cli = ReactomeCli(HttpClient(CIO), "http://reactome.org", "${System.getProperty("user.dir")}/src/test/resources/uniprot.txt")
```

Comment:
I think release.reactome.org might be the test site?

Reply:
Yes, release.reactome.org is considered our staging area if you'd rather not hit production.

Reply:
I think for these tests long term we should probably use https://ktor.io/docs/client-testing.html and have a MockEngine that runs in memory and returns canned responses.